lib/archethic/p2p.ex

```diff
@@ -620,6 +620,7 @@ defmodule Archethic.P2P do
           message :: Message.t(),
           conflict_resolver :: (list(Message.t()) -> Message.t()),
           timeout :: non_neg_integer(),
+          acceptance_resolver :: (Message.t() -> boolean()),
```

Comment:
I think we should move it to the end (last parameter), to be backward compatible

Reply:
I checked we don't use consistency_level anywhere in the project that's why I put it last as it will always take default value